aws-rds-cfn-common/src/main/java/software/amazon/rds/common/client/RdsUserAgentProvider.java

```diff
@@ -0,0 +1,47 @@
+package software.amazon.rds.common.client;
+
+import java.io.IOException;
+import java.io.InputStream;
+import java.util.Properties;
+
+import com.amazonaws.util.StringUtils;
+import software.amazon.rds.common.test.ExcludeFromJacocoGeneratedReport;
+
+public final class RdsUserAgentProvider {
```

Comment:
Is it possible to add date of PR merge or any relatable date so that operator can compare versions based on these dates of changes?

Reply:
No, we can only use information from the SCM (git). A PR, its merge data etc is coming from GitHub, which is a side 3rd party dependency in this context, hence should be avoided as a part of the internal build process.